data.json

```diff
@@ -470,12 +464,6 @@
       },
       "navbar-with-sidebar": {
         "navbar": {
-          "logo": {
```

Comment:
This fixes the "hidden graphic" on the sidebar examples on the components page.

The example was rendering an empty logo as the example did not have one. I ensured that this did not break other sidebars across the site

See #388